test/stmt/if_unexpected_else.swift

```diff
@@ -0,0 +1,5 @@
+// RUN: %target-typecheck-verify-swift
```

Comment:
Nit: Here you shouldn't need the full force of Sema.  Perhaps `target-parse-verify-swift`